dashboard-frontend/src/pages/AccountsPage/AccountList.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { useState } from 'react';
+import { AccountResource } from '../../types/account';
+import { arrayMerge } from '../../helpers/arrays';
+import PrimaryButton from '../../components/PrimaryButton';
+import InputSecret from './InputSecret';
+import InputToken from './InputToken';
+import useCreateAccount from './useCreateAccount';
+import useUpdateAccount from './useUpdateAccount';
+
+const staticAccountResources = [
```

Comment:
Im assuming these are all the account types we support?

Reply:
Exactly, except test!